src/services/replyService.js

```diff
@@ -21,9 +22,12 @@ class ReplyService {
   fetchAllPostReplies() {
     return PostModel.findById(this.postId) //find post
       .then(postRecord => {
-        // store post info...
-        this.postAuthor = postRecord.userId;
         this.postPermLink = postRecord.permLink;
+        return UserModel.findById(postRecord.userId);
+      })
+      .then(userRecord => {
+        // store post info...
+        this.postAuthor = userRecord.name;
```

Comment:
Is there a reason we have to set `postAuthor` as a class variable? This is a service so there really shouldn't be anything attached to the service

Reply:
I believe this was fixd in the previous pr =)